GoogleForCreators/web-stories-wp
# Disable video transcoding for non-WP media

## Context



## Summary

Add `isExternal` attribute to resource not hosted by WordPress. This includes, hotlinks assets, coverr, unsplash and tenor. 
Add a migration for existing image/gif/video elements, to add this new attribute. 

## Relevant Technical Choices



## To-do



## User-facing changes

#### Before 
<img width="298" alt="Screenshot 2021-09-13 at 23 19 38" src="https://user-images.githubusercontent.com/237508/133164060-d3da2398-ae97-4fc4-9f6a-f59d2fa8ddc9.png">

#### After 
<img width="299" alt="Screenshot 2021-09-13 at 23 21 08" src="https://user-images.githubusercontent.com/237508/133164071-fa920018-175d-4194-8571-fa095b990318.png">


## Testing Instructions






This PR can be tested by following these steps:

1. Open editor.
2. Add hotlink video into a page - https://vjs.zencdn.net/v/oceans.mp4
3. Ensure that mute / trim options are not visual. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8982
